src/test/java/Homework16.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest{
+
+    @Test
+    public static void registrationNavigationTest() {
```

Comment:
Please, add Homework16 in TestNG.xml to run tests

![image](https://user-images.githubusercontent.com/104761482/233764961-9b1eb058-e6b0-448d-aeb3-2a53b56f795c.png)

Reply:
done